### ISSUE TITLE
SugarORM encrypts information stored in the DATABASE??

### ISSUE THREAD

**Issue by jorgepilo** [satyan/sugar#685]

**Comment by JakeSteam** [comment]
Nope, not currently. 3-4 years ago [work was started on it](https://github.com/chennaione/sugar/tree/sugar-cipher), and I eventually managed to get it work, but performance was absolutely awful.
